packages/math/texlike.lua

```diff
@@ -362,6 +412,26 @@ compileToMathML(nil, {}, convertTexlike(nil, {[==[
   \def{frac}{\mfrac{#1}{#2}}
   \def{bi}{\mi[mathvariant=bold-italic]{#1}}
   \def{dsi}{\mi[mathvariant=double-struck]{#1}}
+
+  % Standard spaces gleaned from plain TeX
+  \def{thinspace}{\mspace[width=thin]}
+  \def{negthinspace}{\mspace[width=-thin]}
+  \def{,}{\thinspace}
+  \def{!}{\negthinspace}
+  \def{medspace}{\mspace[width=med]}
+  \def{negmedspace}{\mspace[width=-med]}
+  \def{>}{\medspace}
+  \def{thickspace}{\mspace[width=thick]}
+  \def{negthickspace}{\mspace[width=-thick]}
+  \def{;}{\thickspace}
+  \def{enspace}{\mspace[width=1en]}
+  \def{enskip}{\enspace}
+  \def{quad}{\mspace[width=1em]}
```

Comment:
Just to be sure: these are only in math context and won't override the `\quad`, `\qquad`, `\enskip` etc. commands from the plain classes (that are available in non-math context), right?

Reply:
Yes, those are TeX-like macros, they are only defined in the context of TeX-ilke `math`.

Reply:
You rock, folks!